crates/libtiny_tui/src/tui.rs

```diff
@@ -626,6 +643,55 @@ impl TUI {
         tab.set_cursor(cursor);
     }
 
+    /// Shows "mentions" tab at the beginning of the tab list
+    pub(crate) fn show_mentions_tab(&mut self) -> Option<usize> {
```

Comment:
The issue is mainly because we can now /close the mentions tab and it can be recreated in `maybe_create_tab` when you are mentioned...Now that I think about it, it might just be better to not allow /close for the mentions tab. Thoughts?

Reply:
I just tried and I cannot close mentions tab in this branch. It seems like it's accidentally working?

I think it makes sense to not allow closing mentions tab because we rely on having at least one tab available in a few places, and changing all those places will require more design (for example, how to enter commands after closing all tabs) and implementation work.